src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    for (let i = 0; i < this.length; i++) {
+      for (let k = 0; k < this.length; k++) {
+        const comparedEl = this[i];
+        const currentEl = this[k];
+
+        if (compareFunction === undefined) {
+          if (this[i].toString() < this[k].toString()) {
+            this[i] = currentEl;
+            this[k] = comparedEl;
+          }
+
+          continue;
```

Comment:
The default comparison logic should convert elements to strings and compare their UTF-16 code units values. However, it should use `String(a)` instead of `a.toString()` to avoid issues with `null` and `undefined` values. Also, the comparison should be based on `>` instead of `<` to sort in ascending order.